infra/base-images/base-runner/test_all

```diff
@@ -24,12 +24,21 @@ for FUZZER_BINARY in $(find $OUT/ -executable -type f); do
   fi
 
   FUZZER=$(basename $FUZZER_BINARY)
-  if echo "$FUZZER" | grep "^afl-" > /dev/null 2>&1; then
+  if [[ "$FUZZER" == afl-* ]]; then
```

Comment:
From man bash (the bit about [[)

      string1 == string2
       string1 = string2
              True  if  the  strings are equal.  = should be used with the test command for
              POSIX conformance.  When used with the [[ command, this performs
              pattern matching as described above (Compound Commands).

     [[expression]]
       When the == and != operators are used, the string to the right of the operator is considered
        a pattern and matched according to the  rules  described
       below  under  Pattern Matching, as if the extglob shell option were enabled...